internal/cli/workspace_inspect.go

```diff
@@ -0,0 +1,118 @@
+package cli
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/dustin/go-humanize"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	"github.com/posener/complete"
+)
+
+type WorkspaceInspectCommand struct {
+	*baseCommand
+
+	flagWorkspaceName string
+}
+
+func (c *WorkspaceInspectCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	flagSet := c.Flags()
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(flagSet),
+		WithNoConfig(),
+	); err != nil {
+		return 1
+	}
+	args = flagSet.Args()
+
+	if len(args) > 1 {
+		c.ui.Output(c.Help(), terminal.WithErrorStyle())
+		return 1
+	}
+
+	// if a workspace name is given as an argument, use that. This will take
+	// precedence over the -workspace flag as it's an argument to the command
+	var workspaceName string
+	if len(args) > 0 {
+		workspaceName = args[0]
+	}
+
+	if workspaceName == "" {
+		// lookup the default
+		wp, err := c.workspace()
+		if err != nil {
+			c.ui.Output(
+				clierrors.Humanize(errors.New("error loading context default workspace")),
+				terminal.WithErrorStyle(),
+			)
+			return 1
+		}
+		workspaceName = wp
+	}
+
+	workspace, err := getWorkspace(c.Ctx, c.project.Client(), workspaceName)
+	if err != nil {
+		c.ui.Output(
+			clierrors.Humanize(err),
+			terminal.WithErrorStyle(),
+		)
+		return 1
+	}
+
+	var projects []string
+	for _, wp := range workspace.Projects {
+		projects = append(projects, wp.Project.Project)
+	}
+
+	c.ui.Output("Workspace Info:", terminal.WithHeaderStyle())
+
+	var lastActiveTime string
+	if t, err := ptypes.Timestamp(workspace.ActiveTime); err == nil {
+		lastActiveTime = humanize.Time(t)
+	}
+
+	c.ui.NamedValues([]terminal.NamedValue{
+		{
+			Name: "Name", Value: workspace.Name,
+		},
+		{
+			Name: "Latest activity", Value: lastActiveTime,
```

Comment:
It'd be cool if we could actually say what the latest activity was, this looks to be a timestamp but the actual activity was unknown, right? Not sure if we hold onto what the latest operation was. Maybe this header could be `"Last Updated"` since latest activity makes me think I should be getting what the activity was rather than when it happened

Reply:
> the actual activity was unknown, right?

To my knowledge yes, it is unknown. I agree and updated the label used here to be `Last Updated` in ab3cda44